charts/newrelic-infrastructure-v3/templates/kubelet/daemonset.yaml

```diff
@@ -51,9 +51,9 @@ spec:
           image: "{{ .Values.images.integration.registry }}/{{ .Values.images.integration.repository }}:{{ .Values.images.integration.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.images.integration.pullPolicy }}
           securityContext:
-            {{- toYaml .Values.securityContext | nindent 12 }}
+            {{- .Values.securityContext | mustMergeOverwrite (include "newrelic.compatibility.securityContext" . | fromYaml) | toYaml | nindent 12 }}
           env:
-            - name: "NRI_KUBERNETES_HTTPSERVERPORT"
+            - name: "NRI_KUBERNETES_SINK_HTTP_PORT"
```

Comment:
Is this change intended?

Reply:
yes it is needed after https://github.com/newrelic/nri-kubernetes/pull/320